textproc/asciidoc/Portfile

```diff
@@ -4,13 +4,16 @@ PortSystem          1.0
 
 PortGroup           github 1.0
 
-github.setup        asciidoc asciidoc-py3 9.0.5
+version             9.1.0
+github.setup        asciidoc asciidoc-py3 ${version}
 revision            0
-checksums           rmd160  8940c0c8b830f43c97cf163a28bf1bcef31dd173 \
-                    sha256  71acadb505bc34971cdc78a1555e342dfe6da0f0f78b173d96d0dd8519083487 \
-                    size    1119595
+checksums           rmd160  4fef2148ed93859afe13031e0c953bbe0dabba29 \
+                    sha256  98941741bcef1b172bce735c9393255c095fe88642ac725a43a32afda0d73a4a \
+                    size    1131368
 name                asciidoc
 
+dist_subdir         ${name}/${version}
```

Comment:
Right, but when updating the version, the stealth upgrade procedures do no longer apply, so the `dist_subdir` is unnecessary and should not be set.

Sorry for the confusion.

Reply:
All good - learning as I go, so I appreciate the patience.  It should be fixed in the Portfile now